nav-bar.css

```diff
@@ -0,0 +1,35 @@
+.nav-bar {
+  position: fixed;
+  background: linear-gradient(to right, rgb(255, 135, 255), rgb(185, 42, 185));
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 50px;
+  display: flex;
+  align-items: center;
+}
+
+.center-header {
+  display: flex;
+  flex: 8;
+  justify-content: space-around;
+}
+
+.nav-buttons {
+  display: flex;
+  color: white;
+  flex: 1;
+  justify-items: right;
+  padding-right: 10px;
+  justify-content: space-evenly;
+}
+
+a {
```

Comment:
try to be careful about targeting types, because you might overscope certain styles. this affects projects in the long-run